### DISCUSSION TITLE
Is WatsonWebserver built based on HTTP.SYS technology?

### DISCUSSION THREAD

**Discussion by sqlnew** [dotnet/WatsonWebserver#110] [General]
I would like to know what technology is used to implement the underlying Watson Web server. It should support high concurrency. Is it based on Microsoft's HTTP. SYS technology?

**Comment by jchristn** [comment]
Yes, it is a wrapper around the capabilities provided by http.sys.

**Reply by sqlnew** [reply]
This component is very suitable for use in small software, and can achieve the highest core performance of Microsoft. It is very meaningful. I hope to continue to improve and enhance it. Thank you for your work!

**Reply by jchristn** [reply]
Hi @sqlnew I recently released v6.0 which has an http.sys-dependent variant (Watson) and one that does not use http.sys (Watson.Lite).  Cheers :)
